learntools/embeddings/nbs/4-tsne.ipynb

```diff
@@ -383,7 +394,9 @@
     "\n",
     "And just to reiterate: we never actually showed the model genre as a feature. It can't read the titles to see that *Harry Potter and the Philosopher's Stone* and *Harry Potter and the Chamber of Secrets* belong to the same series. It managed to pick up these latent patterns and incorporate them into the embedding space just by seeing data points like \"user 5299 gave movie 806 a rating of 4.5\". Pretty impressive!\n",
     "\n",
-    "Here's another, slightly more complicated genre experiment: visualizing all movies whose genres are a subset of `{Comedy, Drama, Romance}` (i.e. comedies, dramas, romances, dramedies, romantic dramas, romcoms, and... \"dromcoms\" I guess?)"
+    "Here's another, slightly more complicated genre experiment: visualizing all movies whose genres are a subset of `{Comedy, Drama, Romance}` (i.e. comedies, dramas, romances, dramedies, romantic dramas, romcoms, and... \"dromcoms\" I guess?)\n",
+    "\n",
+    "#### DB: I don't think we can assume users know itertools. But I also think we need to justify to the user what reusable skill they are learning here if we are going to keep it."
```

Comment:
This code could definitely use a couple comments explaining at a high level what it's doing. My hope would be that, once those are in place, the reader can either 1) trust that this is doing what I say it's doing and move on, or 2) look up the docs for `itertools.combinations` if they're curious about the low-level details and not familiar with itertools.